spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/AbstractConnectionFactory.java

```diff
@@ -556,6 +559,17 @@ public boolean hasPublisherConnectionFactory() {
 		return this.publisherConnectionFactory != null;
 	}
 
+	/**
+	 * Supports the injection of a backoff strategy during connection creation to accommodate custom retry policies in
```

Comment:
Method Javadocs must be imperative: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting